ui/pages/send/send-content/send-content.component.js

```diff
@@ -50,10 +59,13 @@ export default class SendContent extends Component {
         <div className="send-v2__form">
           {gasError && this.renderError(gasError)}
           {isEthGasPrice && this.renderWarning(ETH_GAS_PRICE_FETCH_WARNING_KEY)}
-          {error && this.renderError()}
+          {unsendableAssetError && this.renderError(UNSENDABLE_ASSET_ERROR_KEY)}
```

Comment:
@darkwing this error will only occur if a user enters the send flow with an NFT token added before these changes were introduced, because the token will not show up as available to send if it has been added with these changes in place (whether it is an ERC721 token is detected upon add here). If you want to test this functionality and don't have any nft tokens previously added, you can comment out line 410 in Preference.js - `newEntry.isERC721 = await this._detectIsERC721(newEntry.address)` in addToken function. Then add an NFT token and then enter the send flow and attempt to select that token to send. Let me know if this makes sense and/or if you want to walk through it together?